tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/openedx-common-settings

```diff
@@ -1,10 +1,10 @@
-{% if K8S_HARMONY_ENABLE_SHARED_ELASTICSEARCH %}
-ELASTICSEARCH_INDEX_PREFIX = "{{ELASTICSEARCH_INDEX_PREFIX}}"
+{% if K8S_HARMONY_ENABLE_SHARED_HARMONY_SEARCH %}
+ELASTICSEARCH_INDEX_PREFIX = "{{HARMONY_SEARCH_INDEX_PREFIX}}"
 ELASTIC_SEARCH_CONFIG = [{
   "use_ssl": True,
-  "host": "elasticsearch-master.{{K8S_HARMONY_NAMESPACE}}.svc.cluster.local",
+  "host": "harmony-search-cluster-master.{{K8S_HARMONY_NAMESPACE}}.svc.cluster.local",
```

Comment:
The service name that I got was actually `harmony-search-cluster.{{K8S_HARMONY_NAMESPACE}}.svc.cluster.local` instead of `harmony-search-cluster-master.{{K8S_HARMONY_NAMESPACE}}.svc.cluster.local`. After changing this I was able to index a course successfully.

Reply:
Thanks @MoisesGSalas. I tested with these config successfully. Will re-test.

Reply:
Missed change after unification.
Fixed.